Code/IO/src/sitkImageIOUtilities.cxx

```diff
@@ -0,0 +1,89 @@
+/*=========================================================================
+*
+*  Copyright Insight Software Consortium
+*
+*  Licensed under the Apache License, Version 2.0 (the "License");
+*  you may not use this file except in compliance with the License.
+*  You may obtain a copy of the License at
+*
+*         http://www.apache.org/licenses/LICENSE-2.0.txt
+*
+*  Unless required by applicable law or agreed to in writing, software
+*  distributed under the License is distributed on an "AS IS" BASIS,
+*  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+*  See the License for the specific language governing permissions and
+*  limitations under the License.
+*
+*=========================================================================*/
+
+#ifndef sitkImageIOUtilities_h
+#define sitkImageIOUtilities_h
+
+#include "itkImageIOBase.h"
+#include <sstream>
+#include <list>
+
+namespace itk {
+namespace simple {
+namespace ioutils {
+
+std::string PrintRegisteredImageIOs()
```

Comment:
Possibly change signature to:
void PrintRegisteredImageIOs( std::ostream & out )